src/dispatch/static/dispatch/src/components/EscalateButton.vue

```diff
@@ -0,0 +1,21 @@
+<!-- EscalateButton.vue -->
+<template>
```

Comment:
Will this exist anywhere but in a case context? Should it be moved out of components and into `/case`?